src/components/calcite-checkbox/calcite-checkbox.tsx

```diff
@@ -213,7 +213,7 @@ export class CalciteCheckbox implements LabelableComponent, CheckableFormCompome
           onFocus={this.onToggleFocus}
           ref={(toggleEl) => (this.toggleEl = toggleEl)}
           role="checkbox"
-          tabIndex={0}
+          tabIndex={this.disabled ? undefined : 0}
```

Comment:
`undefined` should be `-1`, otherwise it will get converted to `0`. The prop expects a number.

Reply:
Do we want disabled inputs to be focusable via clicks?

The intention here is to not set `tabindex` at all when the input is disabled, which is what I think we want in the case of a disabled inputs.  If not, then clicks on the disabled input will focus it, and thus apply focus styles which don't make sense if it is disabled.

Reply:
Ignore my comment, you are right. It looks like Stencil is smart enough to remove the attribute when setting the prop to `undefined`. This is different when working with the raw element itself where the only way to do that was to set the attribute (not prop) to "" or remove it manually (see https://codepen.io/jcfranco/pen/mdBBvvv?editors=1010).